.pre-commit-config.yaml

```diff
@@ -95,6 +95,7 @@ repos:
       WPS102,
       WPS110,
       WPS111,
+      WPS202,
```

Comment:
NB: I disabled this check because it's noisy on what (IMO) are reasonable levels of complexity within files: it's unhappy that `oidc-exchange.py` contains `9 > 7` module members, but I think decomposing those members further would make the code harder, not easier to read.

I can re-enable this and figure out a workaround, but I figured I'd leave a rationale here 🙂

Reply:
@woodruffw I prefer rationale as a code comment where it's disabled. Though, ideally, `# noqa`s should be used in place rather than disabling rules globally. But if the complexity level is known to be higher across the project, there's `--max-module-members` that can be increased instead of disabling the rule too: https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/violations/complexity.html#wemake_python_styleguide.violations.complexity.TooManyModuleMembersViolation

Reply:
As for decomposing the members, I don't think it'd make it harder to read. We just have to choose wisely. For example, the helpers for interfacing with GHA (like messages/warnings) could as well be put into a shared module that more scripts would reuse. This wouldn't make it harder to reason about.

Reply:
> For example, the helpers for interfacing with GHA (like messages/warnings) could as well be put into a shared module that more scripts would reuse. This wouldn't make it harder to reason about.

That's true, although in that case I think we need to switch to a whole package structure here, rather than just Python files in the repo root 🙂

Reply:
Yeah, I was going to look into it at some point. Although, having it in the root as the first step isn't half-bad. We may need to set a `PYTHONPATH` or turn the invocation into a `python -m modulename` for it to be importable until the structure is changed.